### PR TITLE
Add permissions for saving/modifying configurations (Issue #80)

### PR DESCRIPTION
**Pull request by otrok7**
Note that all users must have "manage_options" capability, otherwise the system won't let them save, anyway.

### CODE REVIEW

File: bmlt-meeting-list.php [hunk 1]
```diff
@@ -214,6 +215,8 @@ function enqueue_backend_files($hook) {
 				wp_enqueue_script("tooltipster", plugin_dir_url(__FILE__) . "js/jquery.tooltipster.min.js", array('jquery'), "1.2", true);
 				wp_enqueue_script("spectrum", plugin_dir_url(__FILE__) . "js/spectrum.js", array('jquery'), "1.2", true);
 				wp_enqueue_script("chosen", plugin_dir_url(__FILE__) . "js/chosen.jquery.min.js", array('jquery'), "1.2", true);
+				wp_enqueue_script("selectr", plugin_dir_url(__FILE__) . "js/selectr.min.js");
```
Comment (radius314) [inline bmlt-meeting-list.php]: We already have a control that seems to be similar to this.  I'd rather avoid adding "another" javascript/css library.  Check out how "additional meetings" is being used.

Comment (otrok7) [reply bmlt-meeting-list.php]: When the user changes a meeting list definition, he is effectively changing a Wordpress option.  Therefore, he needs the "manage-options" capability.  This is normally associated with admins, but at least on my local server, I can create a user, give him this capabilty, and then he is able to save.

Comment (otrok7) [reply bmlt-meeting-list.php]: Theres a explorative testing tour called "back alleys"...Bread needs it.  I use bread often and intensely, and I never played with "extra meetings"...

Comment (californiasteve) [reply bmlt-meeting-list.php]: I use "extra meetings" on two different bread installs.

**Comment by radius314** [comment]
please add `.vscode` to the `.gitignore` file.

**Comment by radius314** [comment]
also remove all files from `.vscode` path

**Pull request merged**